lighty-modules/integration-tests/src/test/java/io/lighty/modules/southbound/netconf/tests/TopologyPluginsTest.java

```diff
@@ -68,30 +64,25 @@ public class TopologyPluginsTest {
     private CommunityRestConf restConf;
     private LightyModule netconfPlugin;
     @Mock
-    private NetconfClientDispatcher dispatcher;
```

Comment:
Not removed but replaced by NetconfClientFactory. See https://git.opendaylight.org/gerrit/q/NETCONF-1108.

Reply:
Seems to be addressed in `Rework NetconfTopology initialization`.

Reply:
ACK